docs/commands.adoc

```diff
@@ -359,6 +359,54 @@ Similar to `doc get`, if you want to delete more than one document at the same t
 ╰───┴───────────┴─────────┴────────┴───────────────┴───────────╯
 ```
 
+=== `subdoc get`
+```
+> subdoc get --help
+Fetches the value of the provided path in the specified document through the data service
+
+Usage:
+  > subdoc get {flags} <path> (id)
+
+Flags:
+  -h, --help - Display the help message for this command
+  --id-column <String> - the name of the id column if used with an input stream
+  --bucket <String> - the name of the bucket
+  --scope <String> - the name of the scope
+  --collection <String> - the name of the collection
+  --clusters <String> - the clusters which should be contacted
+  --batch-size <Number> - the maximum number of items to batch send at a time
+  -e, --halt-on-error - halt on any errors
+
+Parameters:
+  path <any>: the path(s) to be fetched from the documents
```

Comment:
An example showing multiple paths being used might be useful?

Reply:
Done!